packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -51,9 +51,27 @@ export function QualityBar({ invalid, valid, empty, digits = 1 }) {
 
 	return (
 		<RatioBar.Composition>
-			<QualityInvalidLine value={invalid} percentage={invalidPercentage} t={t} />
-			<QualityEmptyLine value={empty} percentage={emptyPercentage} t={t} />
-			<QualityValidLine value={valid} percentage={validPercentage} t={t} />
+			<QualityInvalidLine
+				onClick={onClick}
+				dataFeature={dataFeature ? dataFeature('invalid') : null}
```

Comment:
plz use an enum/const object instead of strings and export it so it can be used in apps